scripts/feature_tests/upgrade/README.md

```diff
@@ -0,0 +1,22 @@
+# Upgrade script naming scheme
+
+* nodes: ```<i>cp | <j>w; i=1..n, j=0..n```
```

Comment:
Could you maybe mention also that cp stands for controlplane, and w stands for worker ? It is not clear for someone that is not familiar with those short names.

Reply:
Actually this one also duplicated with https://github.com/metal3-io/metal3-dev-env/pull/357/files#diff-2ccc6e188d7155eaf1e4aa37e6138a17L11.

Reply:
done